openforcefield/typing/engines/smirnoff/parameters.py

```diff
@@ -2676,7 +2676,7 @@ def create_force(self, system, topology, **kwargs):
                 #temp_mol.compute_partial_charges(quantum_chemical_method=self._quantum_chemical_method,
                 #                                 partial_charge_method=self._partial_charge_method)
                 temp_mol.compute_partial_charges_am1bcc(toolkit_registry=toolkit_registry)
-
+                temp_mol.compute_wiberg_bond_orders(toolkit_registry=toolkit_registry)
```

Comment:
I don't think we actually want this to be in the `ToolkitAM1BCCHandler`. WBOs will tie in during the assignment of bond parameter values, so this should be in `BondHandler.create_force`. We'll probably need to copy `generate_conformers` to there as well.